payment/key.go

```diff
@@ -115,6 +63,7 @@ func GenerateSecret() (secret string, nonce string, err error) {
 	if err != nil {
```

Comment:
the comment for this function says "// GenerateSecret creates a random public and secret key" - i don't think it generates a public key?

Reply:
Ah, legacy comments 😩  https://github.com/brave-intl/bat-go/pull/347/commits/45db527bad4b80c34668897ff6b215b9877f0356